files/covjson-reader/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "covjson-reader",
+  "repo": "Reading-eScience-Centre/covjson-reader",
+  "files": {
+    "include": ["*.js"]
```

Comment:
So only minified files should be included? This doesn't match the examples in the documentation. What about source maps? They need the unminified file as well, right?